web/v1/bigquery/sql-runner/sql/custom/README.md

```diff
@@ -100,6 +100,30 @@ derived.commit_table(
 
 If `automigrate` is TRUE, tables which don't exist will be created, and new columns will be added to the target table. If FALSE, the query will fail without committing unless the target table exists and columns match exactly.
 
+### Handling schema migrations
+
+The BigQuery loader for the Snowplow pipeline is unique in that it creates a new column for every new schema for an event or entity, rather than just for major version changes.
+
+The model uses a separate stored procedure to solve this problem internally, which might be useful for custom implementations too:
+
+```SQL
+CALL {{.output_schema}}.combine_context_versions('contexts_com_iab_snowplow_spiders_and_robots_1')
+```
+
+The `combine_context_versions` stored procedure will find all fields within across multiple versions of a context column, and create a new table which coalesces each top-level field, except for arrays or structs.
```

Comment:
Minor but might read better as: 'The `combine_context_versions` stored procedure will find all fields within the context across multiple versions of a context column'